src/game.cpp

```diff
@@ -11326,13 +11326,17 @@ void game::place_player( const tripoint &dest_loc )
             add_msg(m_info, _("Label here: %s"), label.c_str());
         }
     }
-
-    std::string signage = m.get_signage( dest_loc );
-    if (!signage.empty()) {
-        add_msg(m_info, _("The sign says: %s"), signage.c_str());
+    if( !u.has_trait( trait_ILLITERATE ) ) {
+        std::string signage = m.get_signage( dest_loc );
+        if( !signage.empty() ) {
+            add_msg( m_info, _( "The sign says: %s" ), signage.c_str() );
+        }
+        if( m.has_graffiti_at( dest_loc ) ) {
+            add_msg( _( "Written here: %s" ), utf8_truncate( m.graffiti_at( dest_loc ), 40 ).c_str() );
```

Comment:
I think we should get rid of this limit of 40 characters:

![default](https://user-images.githubusercontent.com/11132525/39403254-e28def4c-4b87-11e8-98fa-ae077c8f73e7.png)